### PR TITLE
Get all recipients and funds per block of my pool 

### PR DESCRIPTION
**Pull request by CodeSandwich**
Fixes https://github.com/radicle-dev/radicle-contracts/issues/34 and https://github.com/radicle-dev/radicle-contracts/issues/29

### CODE REVIEW

File: contracts/Pool.sol [hunk 1]
```diff
@@ -80,6 +81,11 @@ contract Pool {
         mapping(uint64 => int256) amtDeltas;
     }
 
+    struct ReceiverWeight {
+        address receiver;
+        uint32 weight;
+    }
```
Comment (geigerzaehler) [inline contracts/Pool.sol:87]: Suggestions:
```suggestion
    struct Receiver {
        address address;
        uint32 weight;
    }
```

File: contracts/Pool.sol [hunk 2]
```diff
@@ -176,6 +182,16 @@ contract Pool {
         senders[msg.sender].amtPerBlock = uint192(amount);
     }
 
+    /// @notice Gets the target amount sent on every block from the sender of the message.
+    /// On every block this amount is rounded down to the closest multiple of the sum of the weights
```
Comment (geigerzaehler) [inline contracts/Pool.sol:186]:
```suggestion
    /// The actual amount send on every block differs from the target value. It is is rounded down to the closest multiple of the sum of the weights
```

File: contracts/Pool.sol (hunk 1 above)
Comment (CodeSandwich) [reply contracts/Pool.sol:87]: Good point, but `struct Receiver` is already defined on line 73. It's used to store the complete information about a receiver and I think that it fits this name better than a container for an address-weight pair.

File: contracts/Pool.sol (hunk 2 above)
Comment (CodeSandwich) [reply contracts/Pool.sol]: Fixed

**Review by geigerzaehler — APPROVED** [review]

**Pull request merged**